pixl_dcmd/src/pixl_dcmd/main.py

```diff
@@ -66,7 +63,7 @@ def anonymise_dicom(dataset: Dataset) -> Dataset:
         DICOM_TAG_PROJECT_NAME.creator_string,
     ).value
     project_config = load_project_config(slug)
-    logger.error(f"Received instance for project {slug}")
+    logger.warning(f"Received instance for project {slug}")
```

Comment:
why warning?

Reply:
It's more inline with the other logging messages. `error` feels weird, no?

Reply:
I see that you've updated this to `logger.debug()` in `main`, which makes even more sense!